resources/telemetry/charts/operator/values.yaml

```diff
@@ -47,6 +47,25 @@ readinessProbe:
   initialDelaySeconds: 5
   periodSeconds: 10
 
+managedfluentbit:
```

Comment:
Why do we need a `managed` prefix? Isn't it implicitly clear? (I also means the subchart name)